samples/RapidWebViewSdkDocsSite/pages/examples/js-demo.tsx

```diff
@@ -221,9 +220,29 @@ const demoItems = [
     link: "/docs/js-interface/functions#16",
     onClick: () => {
       // @ts-ignore
-      app.uploadFile();
+      app.uploadFile("doc","https://google.com/pagead/form-data/1039234079?gtm=45be48s0v9166926293za200&gcd=13l3l3l3l1l1&dma=0&tag_exp=0&npa=0&frm=0&auid=1101065353.1725344348&uaa=arm&uab=64&uafvl=Chromium%3B128.0.6613.113%7CNot%253BA%253DBrand%3B24.0.0.0%7CGoogle%2520Chrome%3B128.0.6613.113&uamb=0&uam=&uap=macOS&uapv=14.6.1&uaw=0","PUT");
```

Comment:
test file upload link  for demo, working fine as of now. ( will remove if not needed )